tests/BUILD

```diff
@@ -0,0 +1,15 @@
+# Copyright 2019 Pants project contributors (see CONTRIBUTORS.md).
+# Licensed under the Apache License, Version 2.0 (see LICENSE).
+
+python_library(
+  name = 'test_base',
+  sources = 'test_base.py',
+)
+
```

Comment:
To get a useful package/module name for code, it's recommended to locate it below a source root: https://www.pantsbuild.org/setup_repo.html#source-roots

Reply:
I don't see how to do this while keeping the flat folder. Changing it to `tests/python/tests` gets the import `tests.test_base` working, but I would love to not have to do this because it means a longer command every time (`./pants test tests/python/tests:first_time_install` vs. `./pants test tests:first_time_install`) and this repo will never be multilingual.

I tried playing around with the options from `./pants help-advanced source`, but was confused what they mean.

Is there a way to do this without changing the folder structure at all?